plugins/keepkey/plugin.py

```diff
@@ -335,11 +335,8 @@ def tx_outputs(self, derivation, tx):
                     txoutputtype.script_type = self.types.PAYTOOPRETURN
                     txoutputtype.op_return_data = address.to_script()[2:]
                 elif _type == TYPE_ADDRESS:
-                    if address.kind == address.ADDR_P2PKH:
-                        txoutputtype.script_type = self.types.PAYTOADDRESS
-                    else:
-                        txoutputtype.script_type = self.types.PAYTOSCRIPTHASH
-                    txoutputtype.address = address.to_string(address.FMT_LEGACY)
+                    txoutputtype.script_type = self.types.PAYTOADDRESS
+                    txoutputtype.address = address.to_full_string(address.FMT_UI)
```

Comment:
Can you just do `address.to_full_ui_string()` here please? I know it seems insignificant but this cuts down on the number of different ways that the Address.to_*_string() is used externally and allows us to change the class more easily later, etc.